templates/openshift/template.yml

```diff
@@ -0,0 +1,14 @@
+skeleton_path: ../skeleton.json
+
+elasticsearch_template:
+  name: com.redhat.viaq-openshift
+  index_pattern: "*"
```

Comment:
Work around for openshift right now, but users of this might need to change this to a specific pattern for their needs.